src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeEndAppealPersonalisation.java

```diff
@@ -67,4 +84,11 @@ public String getReferenceId(Long caseId) {
             .put("homeOfficeReferenceNumber", asylumCase.read(AsylumCaseDefinition.HOME_OFFICE_REFERENCE_NUMBER, String.class).orElse(""))
             .build();
     }
+
+    public boolean isAppealListed(AsylumCase asylumCase) {
```

Comment:
can be private scope? `isAppealListed` method is not a part of interface

Reply:
I've updated all these methods to have package visibility, as the method is also called from within the unit tests. Specifying 'private' access modifier causes a problem with the unit tests.

```
    @Test
    public void should_return_false_if_appeal_not_yet_listed() {
        assertFalse(homeOfficeEndAppealPersonalisation.isAppealListed(asylumCase));
```

Reply:
default (package) is ok too